src/database/sqlite.rs

```diff
@@ -41,6 +41,9 @@ static MIGRATIONS: &[&str] = &[
     "INSERT INTO transaction_details SELECT txid, timestamp, received, sent, fee, height FROM transaction_details_old;",
     "DROP TABLE transaction_details_old;",
     "ALTER TABLE utxos ADD COLUMN is_spent;",
+    "DELETE FROM utxos WHERE rowid > (SELECT MIN(rowid) FROM utxos u2 WHERE utxos.txid = u2.txid AND utxos.vout = u2.vout);",
```

Comment:
I discussed a bit with @afilini about this, while theoretically the duplicated lines should contain the same info, we can't be so sure about that...
If the duplicated lines contain different info (for example: same `outpoint`, different `is_spent`), we might end up in an inconsistent state.
Alekos suggested wiping the entire db.
I suggested wiping it only if we find some duplicated outpoints with the different info - if this can be done in a single SQL query (should be doable, but I'm not entirely sure). What do you think?